runelite-client/src/main/java/net/runelite/client/ui/overlay/components/InfoBoxComponent.java

```diff
@@ -71,7 +71,7 @@ public Dimension render(Graphics2D graphics, Point parent)
 		{
 			graphics.drawImage(image,
 				position.x + (BOX_SIZE - image.getWidth()) / 2,
-				SEPARATOR, null);
+					(BOX_SIZE - image.getHeight()) / 2, null);
```

Comment:
What is the purpose of this?

Reply:
The height of some skill icons (like herblore) is half of that of other skills (like attack), so draw them centered instead of 2px from the top of the box.